### DISCUSSION TITLE
Automatic cleanup of dangling docker images induced by updated egg containers

### DISCUSSION THREAD

**Discussion by DerLev** [pelican-dev/panel#54] [Ideas]
I have had some issues in the past with dangling docker images filling up my storage on my wings host. An automatic task for cleaning up those dangling images would be nice. A current workaround is to set a cron job, which works fine but if this was to be included in the daemon itself or just as a simple note in the docs it would greatly help new users setting things up

**Comment by lancepioch** [comment]
Thanks, please follow #250
